edx/portal-designer
# feat!: Upgrading to django4.2

**Upgrading to django42 and dropped django32 support.**
- PR created under the issue https://github.com/edx/portal-designer/issues/244
- [x]  verified **stage** and **prod** are using `mysql8`. Its `django42` main requirment.
- [x]  make migrations executed, no new migration came
- [x] Removed django32 from tox and gha.
- [ ] Owning team will test and merge.

**Owning team or SRE can confirm mysql8.**


**Testing instruction**

1. Due to sha1 removal all existing signin users will signout. So before deploying on stage login on portal designer site and after upgrading to django42 please check the behaviour. It will ask for login.
